tango_simlib/tango_yaml_tools/base.py

```diff
@@ -0,0 +1,89 @@
+#########################################################################################
+# Copyright 2020 SKA South Africa (http://ska.ac.za/)                                   #
+#                                                                                       #
+# BSD license - see LICENSE.txt for details                                             #
+#########################################################################################
+"""Module that contains the TangoToYAML class that parses a Tango device specification
+   file (xmi, fgo) or a running Tango device into YAML"""
+from pathlib import Path
+
+import yaml
+
+
+class TangoToYAML:
```

Comment:
To still support Python 2.7:
```suggestion
class TangoToYAML(object):
```

Reply:
@ajoubertza The CI pipeline installs and executes tests in both Python2 and Python3 environments.

Looks fine in 2.7.

```
Python 2.7.17 (default, Apr 15 2020, 17:20:14) 
[GCC 7.5.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> class O:
...     pass
... 
>>> o = O()
>>> o
<__main__.O instance at 0x7fd44122bf00>
```

Reply:
Interesting. Didnt know that works in py2.7